pkgs/tools/admin/ansible/default.nix

```diff
@@ -1,9 +1,27 @@
 { python3Packages, fetchurl }:
 
 {
-  ansible = with python3Packages; toPythonApplication ansible;
+  ansible_2_9 = with python3Packages; toPythonApplication ansible;
 
-  ansible_2_8 = with python3Packages; toPythonApplication ansible;
+  ansible = with python3Packages; toPythonApplication (ansible.overridePythonAttrs(old: rec {
```

Comment:
why don't we use an recursive attribute-set and say ansible = ansible_2_8; ?

Reply:
bump @jonringer